### PR TITLE
Direct User notification and subscriber editing

### PR DESCRIPTION
**Pull request by pgrabowski-wikia**
+ Directly notifying e-mail addresses upon scheduled maintenance creation
+ Directly notifying e-mail addresses upon incidents update
+ Admin editing Subscriber e-mails and their subscriptions

### CODE REVIEW

**Comment by GrahamCampbell** [comment]
Thanks for the PR. Please rebase, squash, and fix the tests.

File: config/setting.php [hunk 1]
```diff
@@ -66,7 +66,7 @@
     |
     */
 
-    'show_support' => true,
+    'show_support' => false,
```
Comment (jbrooksuk) [inline config/setting.php]: Please revert this change. You can set this in the `settings` table.

File: .gitignore [hunk 2]
```diff
@@ -2,3 +2,5 @@
 node_modules
 phpunit.xml
 vendor
+.idea
```
Comment (jbrooksuk) [inline .gitignore]: Please revert this change, this should be set in your local `.gitignore` file.

File: .gitignore [hunk 3]
```diff
@@ -2,3 +2,5 @@
...
+composer.lock
```
Comment (jbrooksuk) [inline .gitignore]: Please revert this change, the `composer.lock` file is always committed so that all installations have the same installed dependencies.

File: afterdeploy [hunk 4]
```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
```
Comment (jbrooksuk) [inline afterdeploy]: Please remove this file. This should be part of your deployment.

File: resources/lang/pl/cachet.php [hunk 5]
```diff
@@ -14,10 +14,10 @@
     'components' => [
         'last_updated' => 'Ostatnia aktualizacja :timestamp',
         'status'       => [
-            1 => 'Funktionsfähig',
-            2 => 'Leistungsprobleme',
-            3 => 'Teilweiser Ausfall',
-            4 => 'Schwerer Ausfall',
+            1 => 'Działa',
```
Comment (jbrooksuk) [inline resources/lang/pl/cachet.php]: Please submit these changes to https://translate.cachethq.io

File: resources/views/dashboard/incidents/add.blade.php [hunk 6]
```diff
@@ -123,6 +123,13 @@
                         </label>
                     </div>
                     @endif
+                    @if(direct_notifications_enabled())
+                        <div class="form-group">
```
Comment (jbrooksuk) [inline resources/views/dashboard/incidents/add.blade.php]: Don't need to indent this.

File: resources/views/dashboard/subscribers/edit.blade.php [hunk 7]
```diff
@@ -0,0 +1,72 @@
...
+                <label>
+                    <input type="hidden" value="0" name="is_verified">
+                    <input type="checkbox" value="1" name="is_verified" @if($subscriber->is_verified)checked @endif>
+                    {{ trans("cachet.subscriber.is_verified") }}
+                </label>
+            </div>
+            </fieldset>
+
+            @if(!$components_in_groups->isEmpty() || !$components_out_groups->isEmpty())
+                <hr>
```
Comment (jbrooksuk) [inline resources/views/dashboard/subscribers/edit.blade.php]: Fix the indentation.

File: resources/views/dashboard/subscribers/edit.blade.php [hunk 8]
```diff
@@ -0,0 +1,72 @@
...
+                <span class="uppercase">
+                    <i class="ion ion-ios-browsers-outline"></i> {{ trans('dashboard.components.components') }}
+                </span>
+                <hr>
+                <?php $subscriber_subscriptions = $subscriber->subscriptions->lists("component_id", "component_id"); ?>
```
Comment (jbrooksuk) [inline resources/views/dashboard/subscribers/edit.blade.php]: I'm not keen on this.

File: config/setting.php (hunk 1 above)
Comment (pgrabowski-wikia) [inline config/setting.php]: done

File: .gitignore (hunk 2 above)
Comment (pgrabowski-wikia) [inline .gitignore]: done

File: afterdeploy (hunk 4 above)
Comment (pgrabowski-wikia) [inline afterdeploy]: done

File: resources/lang/pl/cachet.php (hunk 5 above)
Comment (pgrabowski-wikia) [inline resources/lang/pl/cachet.php]: reverted

File: resources/views/dashboard/incidents/add.blade.php (hunk 6 above)
Comment (pgrabowski-wikia) [inline resources/views/dashboard/incidents/add.blade.php]: done

File: resources/views/dashboard/subscribers/edit.blade.php (hunk 7 above)
Comment (pgrabowski-wikia) [inline resources/views/dashboard/subscribers/edit.blade.php]: done

**Comment by pgrabowski-wikia** [comment]
rebased and squashed in https://github.com/CachetHQ/Cachet/pull/2164
